app/routes/home.tsx

```
@@ -1,43 +1,101 @@
+            "Login to upload your resume and get AI-powered feedback."
+          ) : !loadingResumes && resumes?.length === 0 ? (
+            "No resumes found. Upload your first resume to get feedback."
+          ) : (
+            "Review your submissions and check AI-powered feedback."
+          )}
+        </h2>
       </div>
+      
+      {loadingResumes && (
+          <div className="flex flex-col items-center justify-center">
+            <img src="/images/resume-scan-2.gif" className="w-[200px]" />
```

Biome (2.1.2)

[error] Provide a text alternative through the alt, aria-label or aria-labelledby attribute

Meaningful alternative text on elements helps users relying on screen readers to understand content's purpose within a page.
If the content is decorative, redundant, or obscured, consider hiding it from assistive technologies with the aria-hidden attribute.

(lint/a11y/useAltText)